README.rst

```diff
@@ -0,0 +1,113 @@
+FIXME - testing correctness of these links
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
```

Comment:
how do we get this image to work? does some extra step need to be taken with travis? @CJ-Wright @sbillinge

Reply:
I htink you need a newline after the fixme